.github/workflows/CompatHelper.yml

```diff
@@ -2,48 +2,50 @@
 name: CompatHelper
 
 on:
-  schedule:
-    - cron: 0 0 * * *
-  workflow_dispatch:
+    schedule:
+        - cron: 0 0 * * * # Every day at 00:00 UTC
```

Comment:
Seems unnecessary and too complex for a simple template. Notice that this would need to be implemented using Jinja. Also, it would be harder to update, because it would create conflicts.

Reply:
I think you misunderstood me. What I'm saying is, whenever the template is used for the first time, it can randomise the time & day. Not for subsequent updates. So no possibility of conflicts.

Basically, you don't want all sorts of actions on different projects running at the same time just because it happened to use this template. That said, maybe this isn't as important since all this action does is update versions.

Also, IIRC, a jinja template can call custom functions relatively easily. I think you have to import it in the render environment for a function to be available.

Anyway, not a major comment, but something to double check.

Reply:
This would fall in the "cool but not essential feature" category. CompatHelper is already used by most Julia packages, and this was copied from their official repo, which is where people get it from (or from PkgTemplates, that also got it from here). My guess is that GitHub has an auto-scaler for this hour. That said, it would be cool to have something like that. Probably, if you randomize a single number, it will be easy to do with Jinja without importing anything (which is preferable). It's mostly not relevant for this, though, in the sense that it shouldn't be blocking.